Cargo.toml

```diff
@@ -19,82 +19,82 @@ homepage = "https://rerun.io"
 include = ["../../LICENSE-APACHE", "../../LICENSE-MIT", "**/*.rs", "Cargo.toml"]
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/rerun-io/rerun"
-rust-version = "1.72"
-version = "0.13.0-alpha.3"
+rust-version = "1.74"
+version = "0.14.0-alpha.1+dev"
```

Comment:
is it expected that this version is different from the crate versions?

Reply:
No, this is wrong. They should match.

Reply:
Which version are we talking about? Rust's or the crate's?

Reply:
`version = "0.14.0-alpha.1+dev"`

Reply:
Rationale for the missing `+dev` here: https://github.com/rerun-io/rerun/commit/9c7b8d26837214d0110f8331ac63142db09576a2